vibration/lib/vibration.dart

```diff
@@ -14,7 +14,7 @@ class Vibration {
   ///   Vibration.vibrate();
   /// }
   /// ```
-  static Future<bool> hasVibrator() => _channel.invokeMethod("hasVibrator");
+  static Future<bool?> hasVibrator() => _channel.invokeMethod("hasVibrator");
```

Comment:
Would it make sense to treat a `null` value as `false` in terms of the API surface?
Or does a `null` value communicate something different from a `false` value?

Having these `bool` values as optional is a bit confusing from the user's perspective, it would be nicer if they were non-null `bool`s

Reply:
You mean by adding `?? false` at the end? Could it be that the platform channel doesn't work and returns null, but the device has a vibrator? In this case null would be better, so the developers can add `?? false` in their code, or treat `null` differently. But I agree, usually if those values are null, you treat them like they're false.